src/encoded/static/components/static-pages/placeholders/PageCarousel.js

```diff
@@ -0,0 +1,208 @@
+        'autoPlay': false, //bool: starts slide show automatically. not working yet due to a bug.
+        'pauseOnHover': true,
+        'wrapAround': true,
+        'dragging': false,
+        'navControlPosition': 'outside', //left and right nav controls position. Possible values: 'outside' or 'inside'
+        'slideHeight': 240, //integer: Slide height in px. If not defined default is 240.
+        'adjustImageHeight': true, //boolean: if true, resizes slide image's height to container. best for single display power point slides, otherwise set it as false (recommended)
+        'showSlideCount': true, //boolean: displays [Slide x of n] at the bottom left corner. Valid if slidesToShow is defined as 1
+        'textPosition': 'bottom', //'bottom/top/left/right': position of title & description on slide
+    };
+
+    static propTypes = {
+        'slides': PropTypes.array.isRequired,
+        'slidesToShow': PropTypes.number.isRequired,
+        'slidesToScroll': PropTypes.number.isRequired,
+        'slideIndex': PropTypes.number.isRequired,
+        'autoPlay': PropTypes.bool.isRequired,
+        'pauseOnHover': PropTypes.bool.isRequired,
+        'dragging': PropTypes.bool.isRequired,
+        'windowWidth': PropTypes.number,
+        'navControlPosition': PropTypes.string,
+        'slideHeight': PropTypes.number.isRequired,
+        'adjustImageHeight': PropTypes.bool.isRequired,
+        'showSlideCount': PropTypes.bool.isRequired,
+        'textPosition': PropTypes.bool.isRequired,
+    };
+
+    static refFunc(elem) {
+        setTimeout(() => {
+            if (!elem) return;
+            elem.style.opacity = 1;
+        }, 10);
+    }
+
+    /**
+     * this function renders a slide and customize it based on title/description/badge/link and image
+     * @param {Object} slide    Carousel slide to be displayed
+     */
+    renderSlide(slide) {
+        const { img, description, title, badge, badgeBgColor, link } = slide;
+        const { slideHeight, adjustImageHeight, textPosition } = this;
+
+        //todo: move css rules to _static-pages.scss after usability tests
+        let titleContainerStyle = null, badgeOuterStyle = null;
+        if (textPosition === 'bottom') {
+            titleContainerStyle = { left: 0, right: 0, top: 'unset', bottom: 0 };
+            badgeOuterStyle = { left: '10px', right: 'unset' };
+        } else if (textPosition === 'top') {
+            titleContainerStyle = { left: 0, right: 0, top: 0, bottom: 'unset' };
+            badgeOuterStyle = { left: 'unset', right: '10px' };
+        } else if (textPosition === 'left') {
+            titleContainerStyle = { left: 0, right: 'unset', top: 0, bottom: 0, width: '35%' };
+            badgeOuterStyle = { left: 'unset', right: '10px' };
+        } else if (textPosition === 'right') {
+            titleContainerStyle = { left: 'unset', right: 0, top: 0, bottom: 0, width: '35%' };
+            badgeOuterStyle = { left: '10px', right: 'unset' };
+        }
```

Comment:
Will remove this logic until have time to implement with CSS.
Please keep in mind we want the background-images to have `size:  cover` and take into account text size and offset. Will likely need to add "right: 65px" or some similar variant in place of "bottom: 65px", etc.